coding-guides/tracking-used-invites.md

```diff
@@ -61,6 +61,8 @@ client.on('guildMemberAdd', member => {
     logChannel.send(`${member.user.tag} joined using invite code ${invite.code} from ${inviter.tag}. Invite was used ${invite.uses} times since its creation.`);
   });
 });
+
+client.on('inviteCreate', async invite => invites[invite.guild.id] = await invite.guild.fetchInvites())
```

Comment:
I don't see why you would need to fetch all invites here. Would it not be sufficient to do `invites[invite.guild.id] = invite` since you literally just got that data?

Reply:
If you notice I've added this in an `inviteCreate` event and this will only trigger when an invite is created. The reason we're doing this is because or cached invites don't have the new invites. Also if you run your code when the invite count is 0 you get an error.